intro.asciidoc

```diff
@@ -4,36 +4,36 @@
 
-* Private keys can be stored in an encrypted form, as a digital "keystore" file. Being encrypted, they need a password to unlock. When you are prompted to choose a password, make it strong (i.e. long!), back it up and don't share it. If you don't have a password manager, write it down and store it in a safe and secret place. To access your account, you need both the "keystore" file and the password.
+* Private keys can be stored in an encrypted form, as a digital "keystore" file. Being encrypted, they need a password to unlock. When you are prompted to choose a password, make it strong (i.e. long and random), back it up and don't share it. If you don't have a password manager, write it down and store it in a safe and secret place. To access your account, you need both the "keystore" file and the password.
 
-* Do not store any passwords in digital documents, digital photos, screenshots, online drives, encrypted PDFs, etc. Don't improvise security. Use a password manager or pen and paper.
+* Do not store any passwords in digital documents, digital photos, screenshots, online drives, encrypted PDFs, etc. Again, Do not improvise security. Use a password manager or pen and paper.
 
-* When you are prompted to back up a key as a mnemonic word sequence, use pen and paper to make a physical backup. Do not leave that task for "later"; you will forget. These can be used to rebuild your private key in case you lose all data saved on your system, or if you forget or lose your password. However, they can also be used by attackers to get your private keys, and so never store them digitally, and keep the physical copy stored very securely in a locked drawer or safe.
+* When you are prompted to back up a key as a mnemonic word sequence, use pen and paper to make a physical backup. Do not leave that task for "later"; you will forget. These can be used to rebuild your private key in case you lose all data saved on your system, or if you forget or lose your password. However, they can also be used by attackers to get your private keys, and so never store them digitally, and keep the physical copy stored securely in a locked drawer or safe.
 
 * Before transferring any large amounts (especially to new addresses), first do a small test transaction (e.g. less than $1 value) and wait for confirmation of receipt.
 
 * When you create a new account, start by sending only a small test transaction to the new address. Once you receive the test transaction, try sending back again from that account. There are lots of reasons account creation can go wrong, and if it has gone wrong, it is better to find out with a small loss. If sending the test back works, all is well.
 
-* "Block Explorers" are an easy way to independly see whether a transaction has been accepted by the network.
+* Public block explorers are an easy way to independently see whether a transaction has been accepted by the network. However, the easiness comes at the price of privacy.
```

Comment:
Change to: "However, this convenience has a negative impact on your privacy, because you reveal your addresses to block explorers, which can track you."